app/src/interfaces/collection-item-multiple-dropdown/collection-item-multiple-dropdown.vue

```diff
@@ -0,0 +1,246 @@
+		}
+	},
+	{ immediate: true },
+);
+
+function selectionHasChanged(newValues: ValueItem[], oldValues: ValueItem[]) {
+	if (newValues.length !== oldValues.length) return true;
+
+	const newKeys = new Set(newValues.map((v) => `${v.collection}_${v.key}`));
+	const oldKeys = new Set(oldValues.map((v) => `${v.collection}_${v.key}`));
+
+	return ![...newKeys].every((key) => oldKeys.has(key));
+}
+
+async function getDisplayItems() {
+	if (!values.value || values.value.length === 0) {
+		displayItems.value = [];
+		return;
+	}
+
+	if (!unref(selectedCollection) || !primaryKey.value) return;
+
+	const fields = new Set(requiredFields.value);
+	fields.add(primaryKey.value);
+
+	try {
+		const response = await api.get(getEndpoint(unref(selectedCollection)), {
+			params: {
+				fields: Array.from(fields),
+				filter: { [primaryKey.value]: { _in: selectedKeys.value } },
+			},
+		});
+
+		const fetchedItems = response.data.data ?? [];
+
+		displayItems.value = selectedKeys.value
+			.map((key) => fetchedItems.find((item: any) => item[primaryKey.value] === key))
+			.filter(Boolean);
+	} catch (error) {
+		unexpectedError(error);
+	}
+}
+
+function onSelection(selectedIds: (number | string)[] | null) {
+	selectDrawerOpen.value = false;
+
+	if (!selectedIds || selectedIds.length === 0) {
+		values.value = [];
+		return;
+	}
+
+	const newValues = selectedIds.map((key) => ({
+		key,
+		collection: unref(selectedCollection),
+	}));
+
+	values.value = newValues;
+}
+
+function removeItem(item: Record<string, any>) {
+	const keyToRemove = item[primaryKey.value];
+	values.value = values.value.filter((v) => v.key !== keyToRemove);
+}
+
+function onSort(sortedItems: Record<string, any>[]) {
+	displayItems.value = sortedItems;
+
+	const newValues = sortedItems.map((item) => ({
+		key: item[primaryKey.value],
+		collection: unref(selectedCollection),
+	}));
+
+	values.value = newValues;
+}
+</script>
+
+<template>
+	<div class="collection-item-multiple-dropdown">
+		<v-notice v-if="displayItems.length === 0">
+			{{ t('no_items') }}
+		</v-notice>
+
+		<draggable
+			v-else
+			:model-value="displayItems"
+			tag="v-list"
+			item-key="id"
+			handle=".drag-handle"
+			:disabled="disabled"
+			:set-data="hideDragImage"
+			v-bind="{ 'force-fallback': true }"
+			@update:model-value="onSort($event)"
+		>
+			<template #item="{ element }">
+				<v-list-item block :disabled="disabled" :dense="displayItems.length > 4">
+					<v-icon v-if="!disabled" name="drag_handle" class="drag-handle" left @click.stop="() => {}" />
+
+					<render-template
+						:collection="selectedCollection"
+						:item="element"
+						:template="displayTemplate"
+						class="preview"
+					/>
+
+					<div class="spacer" />
+
+					<div class="item-actions">
+						<v-remove v-if="!disabled" deselect @action="removeItem(element)" />
+					</div>
+				</v-list-item>
+			</template>
+		</draggable>
+
+		<div class="actions">
+			<v-button v-if="!disabled" @click="selectDrawerOpen = true">
```

Comment:
Let’s disable the button while keeping it visible.

```suggestion
			<v-button :disabled="disabled" @click="selectDrawerOpen = true">
```